### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Abdona**
Style All Components :

- Navigation bar.
- Book entries.
- Add new.

Use :
- Grid
- Flex
for structuring web components.

### CODE REVIEW

File: src/components/addnewbook.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.addcontainer {
+  margin-top: 3rem;
+  margin-bottom: 10rem;
+}
+
+.addcontainer h1 {
```
Comment (MahmoudBakr23) [inline src/components/addnewbook.css:6]: - [x] Kindly use the same font and color as the one given in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)
Please always keep in mind that following a given design to be EXACTLY as it is, is a very important skill for a software engineer, and here's another chance for you to practice it 😄 🙏 

_**Kindly follow the given design in all aspects, colors, font-size, font-weight, font-family, and spacings (margin and padding)**_

Screenshot (1)

>Please apply the changes required on this point all over the whole page!

File: src/components/bookComponent.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+.book {
+  background-color: white;
+  height: 7.625rem;
+  border: 1px solid #e8e8e8;
+  display: flex;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+}
```
Comment (MahmoudBakr23) [inline src/components/bookComponent.css:13]: - [x] In each book `<div>` element, should have the tracking graph on the right side of the text '65% Completed'. Kindly let's add it as given in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏

File: src/components/navbar.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom';
+import './navbar.css';
+
+const Navbar = () => (
```
Comment (MahmoudBakr23) [inline src/components/navbar.js:5]: - [x] In the Navbar component, please add the icon displayed at the very right-top corner of the header as shown in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello, @Abdona 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct
- Descriptive README file
- Extra credit for the Live Demo Link

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [x] Please write a more descriptive PR headline for a professional PR and also to help viewers catch what this PR is really about easily! 😄 🙏 
_Please note that this is a hard requirement that I highly suggest you pay attention to it in all of your projects to prevent getting your PRs invalidated!_

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Abdona 

Great work on implementing the changes requested by the previous reviewer, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Live link demo is working properly :heavy_check_mark:
- Your project looks awesome :heavy_check_mark:

Changes required :writing_hand: 

       - The `50% completed` has been wrapped by the circle, please change the margin left or right for it.
       
style

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Abdona** [comment]
> ## STATUS: CHANGES REQUESTED 
> Hi, @Abdona
> 
> Great work on implementing the changes requested by the previous reviewer, however there are some issues to be solved before you get approved but you are almost there.
> 
> * The repo looks professional.heavy_check_mark
> * Git/GitHub flow best practices have been followedheavy_check_mark
> * There are no linter errorsheavy_check_mark
> * You have descriptive PRheavy_check_mark
> * Live link demo is working properly heavy_check_mark
> * Your project looks awesome heavy_check_mark
> 
> Changes required 
> 
> ```
>    - The `50% completed` has been wrapped by the circle, please change the margin left or right for it.
> ```
> 
> style
> 
> Cheers and Happy coding!   
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


Thank you for your review @Emmyn5600 
The '50%' Progress is working well on my side.
I have tried it on any browser with no margin problem.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Abdona 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**